<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GasVDefaults</class>
 <widget class="QWidget" name="GasVDefaults">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>213</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QGroupBox" name="gbScale">
     <property name="title">
      <string>Scale</string>
     </property>
     <layout class="QGridLayout">
      <property name="margin">
       <number>9</number>
      </property>
      <property name="spacing">
       <number>6</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="lFGF">
        <property name="text">
         <string>FGF</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QSpinBox" name="sbCO"/>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="lCO">
        <property name="text">
         <string>CO</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="sbVA"/>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="lVA">
        <property name="text">
         <string>VA</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QSpinBox" name="sbFGF"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lView">
     <property name="text">
      <string>View:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QGridLayout" name="gridLayout1">
     <item row="0" column="0">
      <widget class="QComboBox" name="cbTime"/>
     </item>
     <item row="0" column="1">
      <widget class="QCheckBox" name="cbShowCost">
       <property name="text">
        <string>Show Cost</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QCheckBox" name="cbEnableScrolling">
       <property name="text">
        <string>Enable Scrolling</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QCheckBox" name="cbShowMl">
       <property name="text">
        <string>Show mL</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QCheckBox" name="cbShowControl">
       <property name="text">
        <string>Show Control Tray</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>254</width>
       <height>21</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
